QemuPkg/Library/DfciUiSupportLib/DfciUiSupportLib.c

```diff
@@ -0,0 +1,199 @@
+/** @file
+DfciUiSupportLibNull.c
+
+NULL instance of the UiSupportLib.

```

Comment:
If this is the null instance, should it be called DfciUiSupportLibNull? Or, is this comment and line 2 copied from the null instance?

Reply:
Not the NULL.

Reply:
And fixed